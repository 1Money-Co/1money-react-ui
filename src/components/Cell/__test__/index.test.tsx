import 'jsdom-global/register';
import * as React from 'react';
import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import '@testing-library/jest-dom';
import { Cell } from '../index';

const originalConsoleError = console.error;
console.error = (message, ...optionalParams) => {
  if (
    message.includes('Could not parse CSS stylesheet') ||
    message.includes('findDOMNode is deprecated and will be removed') ||
    message.includes('React does not recognize')
  ) {
      return;
  }
  originalConsoleError(message, ...optionalParams);
};

describe('Cell', () => {
  it('renders correctly', () => {
    const defaultRender = render(<Cell />);
    const active = render(<Cell active />);
      const disabled = render(<Cell disabled />);

    expect(defaultRender).toMatchSnapshot();
    expect(active).toMatchSnapshot();
    expect(disabled).toMatchSnapshot();
  });
});
