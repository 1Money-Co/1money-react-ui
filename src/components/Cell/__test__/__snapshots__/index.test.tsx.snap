// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="om-react-ui-cell   "
      >
        <div
          class="om-react-ui-cell-inner"
        >
          <svg
            class="om-react-ui-icons"
            fill="#000000"
            height="24"
            stroke="none"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M4 3H12H20V7.58561C20 13.1323 16.9099 18.2095 12 20.7619C7.09012 18.2095 4 13.1323 4 7.58561V3ZM2 3C2 1.89543 2.89543 1 4 1H12H20C21.1046 1 22 1.89543 22 3V7.58561C22 13.9682 18.3939 19.8031 12.6851 22.6575C12.2538 22.8731 11.7462 22.8731 11.3149 22.6575C5.60612 19.8031 2 13.9682 2 7.58561V3ZM12 8.49976C12.5523 8.49976 13 8.05204 13 7.49976C13 6.94747 12.5523 6.49976 12 6.49976C11.4477 6.49976 11 6.94747 11 7.49976C11 8.05204 11.4477 8.49976 12 8.49976ZM15 7.49976C15 8.80597 14.1652 9.91721 13 10.3291V12.9998H15V14.9998H13V16.9998H11V10.3291C9.83481 9.91721 9 8.80597 9 7.49976C9 5.8429 10.3431 4.49976 12 4.49976C13.6569 4.49976 15 5.8429 15 7.49976Z"
              fill-rule="evenodd"
            />
          </svg>
          <h3
            class="om-react-ui-typography-title om-react-ui-typography-title-l  "
          >
            Authenticator
          </h3>
        </div>
        <svg
          class="om-react-ui-icons"
          fill="none"
          height="24"
          stroke="#000000"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 12L19 12M19 12L12 19M19 12L12 5"
            stroke-width="2"
          />
        </svg>
      </div>
    </div>
    <div>
      <div
        class="om-react-ui-cell om-react-ui-cell-active  "
      >
        <div
          class="om-react-ui-cell-inner"
        >
          <svg
            class="om-react-ui-icons"
            fill="#000000"
            height="24"
            stroke="none"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M4 3H12H20V7.58561C20 13.1323 16.9099 18.2095 12 20.7619C7.09012 18.2095 4 13.1323 4 7.58561V3ZM2 3C2 1.89543 2.89543 1 4 1H12H20C21.1046 1 22 1.89543 22 3V7.58561C22 13.9682 18.3939 19.8031 12.6851 22.6575C12.2538 22.8731 11.7462 22.8731 11.3149 22.6575C5.60612 19.8031 2 13.9682 2 7.58561V3ZM12 8.49976C12.5523 8.49976 13 8.05204 13 7.49976C13 6.94747 12.5523 6.49976 12 6.49976C11.4477 6.49976 11 6.94747 11 7.49976C11 8.05204 11.4477 8.49976 12 8.49976ZM15 7.49976C15 8.80597 14.1652 9.91721 13 10.3291V12.9998H15V14.9998H13V16.9998H11V10.3291C9.83481 9.91721 9 8.80597 9 7.49976C9 5.8429 10.3431 4.49976 12 4.49976C13.6569 4.49976 15 5.8429 15 7.49976Z"
              fill-rule="evenodd"
            />
          </svg>
          <h3
            class="om-react-ui-typography-title om-react-ui-typography-title-l  "
          >
            Authenticator
          </h3>
        </div>
        <svg
          class="om-react-ui-icons"
          fill="none"
          height="24"
          stroke="#000000"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 12L19 12M19 12L12 19M19 12L12 5"
            stroke-width="2"
          />
        </svg>
      </div>
    </div>
    <div>
      <div
        class="om-react-ui-cell  om-react-ui-cell-disabled "
      >
        <div
          class="om-react-ui-cell-inner"
        >
          <svg
            class="om-react-ui-icons"
            fill="#000000"
            height="24"
            stroke="none"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M4 3H12H20V7.58561C20 13.1323 16.9099 18.2095 12 20.7619C7.09012 18.2095 4 13.1323 4 7.58561V3ZM2 3C2 1.89543 2.89543 1 4 1H12H20C21.1046 1 22 1.89543 22 3V7.58561C22 13.9682 18.3939 19.8031 12.6851 22.6575C12.2538 22.8731 11.7462 22.8731 11.3149 22.6575C5.60612 19.8031 2 13.9682 2 7.58561V3ZM12 8.49976C12.5523 8.49976 13 8.05204 13 7.49976C13 6.94747 12.5523 6.49976 12 6.49976C11.4477 6.49976 11 6.94747 11 7.49976C11 8.05204 11.4477 8.49976 12 8.49976ZM15 7.49976C15 8.80597 14.1652 9.91721 13 10.3291V12.9998H15V14.9998H13V16.9998H11V10.3291C9.83481 9.91721 9 8.80597 9 7.49976C9 5.8429 10.3431 4.49976 12 4.49976C13.6569 4.49976 15 5.8429 15 7.49976Z"
              fill-rule="evenodd"
            />
          </svg>
          <h3
            class="om-react-ui-typography-title om-react-ui-typography-title-l  "
          >
            Authenticator
          </h3>
        </div>
        <svg
          class="om-react-ui-icons"
          fill="none"
          height="24"
          stroke="#000000"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 12L19 12M19 12L12 19M19 12L12 5"
            stroke-width="2"
          />
        </svg>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="om-react-ui-cell   "
    >
      <div
        class="om-react-ui-cell-inner"
      >
        <svg
          class="om-react-ui-icons"
          fill="#000000"
          height="24"
          stroke="none"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M4 3H12H20V7.58561C20 13.1323 16.9099 18.2095 12 20.7619C7.09012 18.2095 4 13.1323 4 7.58561V3ZM2 3C2 1.89543 2.89543 1 4 1H12H20C21.1046 1 22 1.89543 22 3V7.58561C22 13.9682 18.3939 19.8031 12.6851 22.6575C12.2538 22.8731 11.7462 22.8731 11.3149 22.6575C5.60612 19.8031 2 13.9682 2 7.58561V3ZM12 8.49976C12.5523 8.49976 13 8.05204 13 7.49976C13 6.94747 12.5523 6.49976 12 6.49976C11.4477 6.49976 11 6.94747 11 7.49976C11 8.05204 11.4477 8.49976 12 8.49976ZM15 7.49976C15 8.80597 14.1652 9.91721 13 10.3291V12.9998H15V14.9998H13V16.9998H11V10.3291C9.83481 9.91721 9 8.80597 9 7.49976C9 5.8429 10.3431 4.49976 12 4.49976C13.6569 4.49976 15 5.8429 15 7.49976Z"
            fill-rule="evenodd"
          />
        </svg>
        <h3
          class="om-react-ui-typography-title om-react-ui-typography-title-l  "
        >
          Authenticator
        </h3>
      </div>
      <svg
        class="om-react-ui-icons"
        fill="none"
        height="24"
        stroke="#000000"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M4 12L19 12M19 12L12 19M19 12L12 5"
          stroke-width="2"
        />
      </svg>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Cell renders correctly 2`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="om-react-ui-cell   "
      >
        <div
          class="om-react-ui-cell-inner"
        >
          <svg
            class="om-react-ui-icons"
            fill="#000000"
            height="24"
            stroke="none"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M4 3H12H20V7.58561C20 13.1323 16.9099 18.2095 12 20.7619C7.09012 18.2095 4 13.1323 4 7.58561V3ZM2 3C2 1.89543 2.89543 1 4 1H12H20C21.1046 1 22 1.89543 22 3V7.58561C22 13.9682 18.3939 19.8031 12.6851 22.6575C12.2538 22.8731 11.7462 22.8731 11.3149 22.6575C5.60612 19.8031 2 13.9682 2 7.58561V3ZM12 8.49976C12.5523 8.49976 13 8.05204 13 7.49976C13 6.94747 12.5523 6.49976 12 6.49976C11.4477 6.49976 11 6.94747 11 7.49976C11 8.05204 11.4477 8.49976 12 8.49976ZM15 7.49976C15 8.80597 14.1652 9.91721 13 10.3291V12.9998H15V14.9998H13V16.9998H11V10.3291C9.83481 9.91721 9 8.80597 9 7.49976C9 5.8429 10.3431 4.49976 12 4.49976C13.6569 4.49976 15 5.8429 15 7.49976Z"
              fill-rule="evenodd"
            />
          </svg>
          <h3
            class="om-react-ui-typography-title om-react-ui-typography-title-l  "
          >
            Authenticator
          </h3>
        </div>
        <svg
          class="om-react-ui-icons"
          fill="none"
          height="24"
          stroke="#000000"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 12L19 12M19 12L12 19M19 12L12 5"
            stroke-width="2"
          />
        </svg>
      </div>
    </div>
    <div>
      <div
        class="om-react-ui-cell om-react-ui-cell-active  "
      >
        <div
          class="om-react-ui-cell-inner"
        >
          <svg
            class="om-react-ui-icons"
            fill="#000000"
            height="24"
            stroke="none"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M4 3H12H20V7.58561C20 13.1323 16.9099 18.2095 12 20.7619C7.09012 18.2095 4 13.1323 4 7.58561V3ZM2 3C2 1.89543 2.89543 1 4 1H12H20C21.1046 1 22 1.89543 22 3V7.58561C22 13.9682 18.3939 19.8031 12.6851 22.6575C12.2538 22.8731 11.7462 22.8731 11.3149 22.6575C5.60612 19.8031 2 13.9682 2 7.58561V3ZM12 8.49976C12.5523 8.49976 13 8.05204 13 7.49976C13 6.94747 12.5523 6.49976 12 6.49976C11.4477 6.49976 11 6.94747 11 7.49976C11 8.05204 11.4477 8.49976 12 8.49976ZM15 7.49976C15 8.80597 14.1652 9.91721 13 10.3291V12.9998H15V14.9998H13V16.9998H11V10.3291C9.83481 9.91721 9 8.80597 9 7.49976C9 5.8429 10.3431 4.49976 12 4.49976C13.6569 4.49976 15 5.8429 15 7.49976Z"
              fill-rule="evenodd"
            />
          </svg>
          <h3
            class="om-react-ui-typography-title om-react-ui-typography-title-l  "
          >
            Authenticator
          </h3>
        </div>
        <svg
          class="om-react-ui-icons"
          fill="none"
          height="24"
          stroke="#000000"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 12L19 12M19 12L12 19M19 12L12 5"
            stroke-width="2"
          />
        </svg>
      </div>
    </div>
    <div>
      <div
        class="om-react-ui-cell  om-react-ui-cell-disabled "
      >
        <div
          class="om-react-ui-cell-inner"
        >
          <svg
            class="om-react-ui-icons"
            fill="#000000"
            height="24"
            stroke="none"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M4 3H12H20V7.58561C20 13.1323 16.9099 18.2095 12 20.7619C7.09012 18.2095 4 13.1323 4 7.58561V3ZM2 3C2 1.89543 2.89543 1 4 1H12H20C21.1046 1 22 1.89543 22 3V7.58561C22 13.9682 18.3939 19.8031 12.6851 22.6575C12.2538 22.8731 11.7462 22.8731 11.3149 22.6575C5.60612 19.8031 2 13.9682 2 7.58561V3ZM12 8.49976C12.5523 8.49976 13 8.05204 13 7.49976C13 6.94747 12.5523 6.49976 12 6.49976C11.4477 6.49976 11 6.94747 11 7.49976C11 8.05204 11.4477 8.49976 12 8.49976ZM15 7.49976C15 8.80597 14.1652 9.91721 13 10.3291V12.9998H15V14.9998H13V16.9998H11V10.3291C9.83481 9.91721 9 8.80597 9 7.49976C9 5.8429 10.3431 4.49976 12 4.49976C13.6569 4.49976 15 5.8429 15 7.49976Z"
              fill-rule="evenodd"
            />
          </svg>
          <h3
            class="om-react-ui-typography-title om-react-ui-typography-title-l  "
          >
            Authenticator
          </h3>
        </div>
        <svg
          class="om-react-ui-icons"
          fill="none"
          height="24"
          stroke="#000000"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 12L19 12M19 12L12 19M19 12L12 5"
            stroke-width="2"
          />
        </svg>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="om-react-ui-cell om-react-ui-cell-active  "
    >
      <div
        class="om-react-ui-cell-inner"
      >
        <svg
          class="om-react-ui-icons"
          fill="#000000"
          height="24"
          stroke="none"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M4 3H12H20V7.58561C20 13.1323 16.9099 18.2095 12 20.7619C7.09012 18.2095 4 13.1323 4 7.58561V3ZM2 3C2 1.89543 2.89543 1 4 1H12H20C21.1046 1 22 1.89543 22 3V7.58561C22 13.9682 18.3939 19.8031 12.6851 22.6575C12.2538 22.8731 11.7462 22.8731 11.3149 22.6575C5.60612 19.8031 2 13.9682 2 7.58561V3ZM12 8.49976C12.5523 8.49976 13 8.05204 13 7.49976C13 6.94747 12.5523 6.49976 12 6.49976C11.4477 6.49976 11 6.94747 11 7.49976C11 8.05204 11.4477 8.49976 12 8.49976ZM15 7.49976C15 8.80597 14.1652 9.91721 13 10.3291V12.9998H15V14.9998H13V16.9998H11V10.3291C9.83481 9.91721 9 8.80597 9 7.49976C9 5.8429 10.3431 4.49976 12 4.49976C13.6569 4.49976 15 5.8429 15 7.49976Z"
            fill-rule="evenodd"
          />
        </svg>
        <h3
          class="om-react-ui-typography-title om-react-ui-typography-title-l  "
        >
          Authenticator
        </h3>
      </div>
      <svg
        class="om-react-ui-icons"
        fill="none"
        height="24"
        stroke="#000000"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M4 12L19 12M19 12L12 19M19 12L12 5"
          stroke-width="2"
        />
      </svg>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Cell renders correctly 3`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="om-react-ui-cell   "
      >
        <div
          class="om-react-ui-cell-inner"
        >
          <svg
            class="om-react-ui-icons"
            fill="#000000"
            height="24"
            stroke="none"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M4 3H12H20V7.58561C20 13.1323 16.9099 18.2095 12 20.7619C7.09012 18.2095 4 13.1323 4 7.58561V3ZM2 3C2 1.89543 2.89543 1 4 1H12H20C21.1046 1 22 1.89543 22 3V7.58561C22 13.9682 18.3939 19.8031 12.6851 22.6575C12.2538 22.8731 11.7462 22.8731 11.3149 22.6575C5.60612 19.8031 2 13.9682 2 7.58561V3ZM12 8.49976C12.5523 8.49976 13 8.05204 13 7.49976C13 6.94747 12.5523 6.49976 12 6.49976C11.4477 6.49976 11 6.94747 11 7.49976C11 8.05204 11.4477 8.49976 12 8.49976ZM15 7.49976C15 8.80597 14.1652 9.91721 13 10.3291V12.9998H15V14.9998H13V16.9998H11V10.3291C9.83481 9.91721 9 8.80597 9 7.49976C9 5.8429 10.3431 4.49976 12 4.49976C13.6569 4.49976 15 5.8429 15 7.49976Z"
              fill-rule="evenodd"
            />
          </svg>
          <h3
            class="om-react-ui-typography-title om-react-ui-typography-title-l  "
          >
            Authenticator
          </h3>
        </div>
        <svg
          class="om-react-ui-icons"
          fill="none"
          height="24"
          stroke="#000000"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 12L19 12M19 12L12 19M19 12L12 5"
            stroke-width="2"
          />
        </svg>
      </div>
    </div>
    <div>
      <div
        class="om-react-ui-cell om-react-ui-cell-active  "
      >
        <div
          class="om-react-ui-cell-inner"
        >
          <svg
            class="om-react-ui-icons"
            fill="#000000"
            height="24"
            stroke="none"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M4 3H12H20V7.58561C20 13.1323 16.9099 18.2095 12 20.7619C7.09012 18.2095 4 13.1323 4 7.58561V3ZM2 3C2 1.89543 2.89543 1 4 1H12H20C21.1046 1 22 1.89543 22 3V7.58561C22 13.9682 18.3939 19.8031 12.6851 22.6575C12.2538 22.8731 11.7462 22.8731 11.3149 22.6575C5.60612 19.8031 2 13.9682 2 7.58561V3ZM12 8.49976C12.5523 8.49976 13 8.05204 13 7.49976C13 6.94747 12.5523 6.49976 12 6.49976C11.4477 6.49976 11 6.94747 11 7.49976C11 8.05204 11.4477 8.49976 12 8.49976ZM15 7.49976C15 8.80597 14.1652 9.91721 13 10.3291V12.9998H15V14.9998H13V16.9998H11V10.3291C9.83481 9.91721 9 8.80597 9 7.49976C9 5.8429 10.3431 4.49976 12 4.49976C13.6569 4.49976 15 5.8429 15 7.49976Z"
              fill-rule="evenodd"
            />
          </svg>
          <h3
            class="om-react-ui-typography-title om-react-ui-typography-title-l  "
          >
            Authenticator
          </h3>
        </div>
        <svg
          class="om-react-ui-icons"
          fill="none"
          height="24"
          stroke="#000000"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 12L19 12M19 12L12 19M19 12L12 5"
            stroke-width="2"
          />
        </svg>
      </div>
    </div>
    <div>
      <div
        class="om-react-ui-cell  om-react-ui-cell-disabled "
      >
        <div
          class="om-react-ui-cell-inner"
        >
          <svg
            class="om-react-ui-icons"
            fill="#000000"
            height="24"
            stroke="none"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M4 3H12H20V7.58561C20 13.1323 16.9099 18.2095 12 20.7619C7.09012 18.2095 4 13.1323 4 7.58561V3ZM2 3C2 1.89543 2.89543 1 4 1H12H20C21.1046 1 22 1.89543 22 3V7.58561C22 13.9682 18.3939 19.8031 12.6851 22.6575C12.2538 22.8731 11.7462 22.8731 11.3149 22.6575C5.60612 19.8031 2 13.9682 2 7.58561V3ZM12 8.49976C12.5523 8.49976 13 8.05204 13 7.49976C13 6.94747 12.5523 6.49976 12 6.49976C11.4477 6.49976 11 6.94747 11 7.49976C11 8.05204 11.4477 8.49976 12 8.49976ZM15 7.49976C15 8.80597 14.1652 9.91721 13 10.3291V12.9998H15V14.9998H13V16.9998H11V10.3291C9.83481 9.91721 9 8.80597 9 7.49976C9 5.8429 10.3431 4.49976 12 4.49976C13.6569 4.49976 15 5.8429 15 7.49976Z"
              fill-rule="evenodd"
            />
          </svg>
          <h3
            class="om-react-ui-typography-title om-react-ui-typography-title-l  "
          >
            Authenticator
          </h3>
        </div>
        <svg
          class="om-react-ui-icons"
          fill="none"
          height="24"
          stroke="#000000"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 12L19 12M19 12L12 19M19 12L12 5"
            stroke-width="2"
          />
        </svg>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="om-react-ui-cell  om-react-ui-cell-disabled "
    >
      <div
        class="om-react-ui-cell-inner"
      >
        <svg
          class="om-react-ui-icons"
          fill="#000000"
          height="24"
          stroke="none"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M4 3H12H20V7.58561C20 13.1323 16.9099 18.2095 12 20.7619C7.09012 18.2095 4 13.1323 4 7.58561V3ZM2 3C2 1.89543 2.89543 1 4 1H12H20C21.1046 1 22 1.89543 22 3V7.58561C22 13.9682 18.3939 19.8031 12.6851 22.6575C12.2538 22.8731 11.7462 22.8731 11.3149 22.6575C5.60612 19.8031 2 13.9682 2 7.58561V3ZM12 8.49976C12.5523 8.49976 13 8.05204 13 7.49976C13 6.94747 12.5523 6.49976 12 6.49976C11.4477 6.49976 11 6.94747 11 7.49976C11 8.05204 11.4477 8.49976 12 8.49976ZM15 7.49976C15 8.80597 14.1652 9.91721 13 10.3291V12.9998H15V14.9998H13V16.9998H11V10.3291C9.83481 9.91721 9 8.80597 9 7.49976C9 5.8429 10.3431 4.49976 12 4.49976C13.6569 4.49976 15 5.8429 15 7.49976Z"
            fill-rule="evenodd"
          />
        </svg>
        <h3
          class="om-react-ui-typography-title om-react-ui-typography-title-l  "
        >
          Authenticator
        </h3>
      </div>
      <svg
        class="om-react-ui-icons"
        fill="none"
        height="24"
        stroke="#000000"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M4 12L19 12M19 12L12 19M19 12L12 5"
          stroke-width="2"
        />
      </svg>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
